pkgs/by-name/ra/rainbowcrack/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, alglib
+, unzip
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8";
+
+  src = fetchurl {
+    url = "http://project-rainbowcrack.com/rainbowcrack-${version}-linux64.zip";
+    hash = "sha256-xMC9teHiDvBY/VHV63TsNQjdcuLqHGeXUyjHvRTO9HQ=";
+  };
+
+  buildInputs = [ alglib unzip ];
```

Comment:
this should be `nativeBuildInputs`, also where is `alglib` being used?

Reply:
it is just a required lib but not actually used to build the package.

Reply:
`buildInputs` will not work in this case. There's no linker command in this derivation. You may need `autoPatchelfHook` with `runtimeDependencies`.

Reply:
changes made stdenNoCC to stdenv, autoPatchelfHook and runtimedependencies added, this way is most preferred Thank you @Aleksanaa